xml/Microsoft.Extensions.Primitives/StringSegment.xml

```diff
@@ -442,9 +442,10 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
+        <param name="obj">An object to compare with this object.</param>
         <summary>Indicates whether the current object is equal to another object of the same type.</summary>
-        <returns>To be added.</returns>
+        <returns>
+          <see langword="true" /> if the current object is equal to the other parameter; otherwise, <see langword="false" />.</returns>
         <remarks>To be added.</remarks>
         <inheritdoc />
```

Comment:
@gewarren do you think the other warning message is caused by having `<inheritdoc />` here?

```
Found no member can be inherited by key:Equals(System.Object) for uid: Microsoft.Extensions.Primitives.StringSegment.Equals(System.Object).
```

Reply:
Is it ok if we remove it, since we're not inheriting docs?

```suggestion
```

Reply:
Yes, it's fine with me.

Reply:
Opened issue to get this fixed with the tool: https://github.com/dotnet/api-docs-sync/issues/116

Reply:
Actually I think we can leave this tag as-is. The suggestion will go away when https://dev.azure.com/ceapex/Engineering/_workitems/edit/567948 is fixed. There's a bug in the ECMA2YML system that only inherits docs from classes, not structs.